routing/router.go

```diff
@@ -1560,10 +1560,8 @@ type LightningPayment struct {
 
 	// FinalCLTVDelta is the CTLV expiry delta to use for the _final_ hop
 	// in the route. This means that the final hop will have a CLTV delta
-	// of at least: currentHeight + FinalCLTVDelta. If this value is
-	// unspecified, then a default value of DefaultFinalCLTVDelta will be
-	// used.
-	FinalCLTVDelta *uint16
+	// of at least: currentHeight + FinalCLTVDelta.
+	FinalCLTVDelta int16
```

Comment:
👍 

Can we do this for the cltv limit also?

Reply:
No, because the cltv limit really is optional, while final cltv is required.

Reply:
Fixed